### DISCUSSION TITLE
Notifiche su webapp ANPR

### DISCUSSION THREAD

**Discussion by SimonaGrisi** [italia/ansc#1096] [Supporto Webapp]
Eseguendo un test sulla WEB App ANSC di test abbiamo riscontrato un comportamento inatteso:
abbiamo caricato un evento di nascita con entrambi i genitori residenti, apparentemente ci sembra di aver eseguito tutti i passaggi per la validazione dell'atto e ci aspettavamo che arrivasse una notifica in ANPR, sempre ambiente di test, che però non è arrivata.
L'atto inserito è il numero 3514534 (id atto)  2024-3514534-1-017074 (id-ansc), potreste indicarci eventuali errori o se abbiamo omesso qualche passaggio?
Si ringrazia anticipatamente per l'aiuto.

**Comment by acolosimosogei** [comment]
Salve, 
il problema delle notifiche è probabilmente dovuto ad una data di adesione impropria registrata per il comune; abbiamo provveduto ad effettuare la correzione da centro. Potete riprovare il caso di test ex novo. 
Attendiamo un vostro feedback.
Grazie.

**Comment by acolosimosogei** [comment]
Buonasera,
procedo a chiudere la discussion, in caso di aggiornamenti può essere riaperto in ogni momento.
Grazie,
Assistenza ANSC
